docs/source/reference/api-server/api-server-admin-deploy.rst

```diff
@@ -185,13 +69,15 @@ The ``--namespace`` flag specifies which namespace to deploy the API server in,
 
 To install a specific version, pass the ``--version`` flag to the ``helm upgrade`` command (e.g., ``--version 0.1.0``).
 
-If you configured any cloud credentials in the previous step, make sure to enable them by adding the relevant flags (e.g., ``--set awsCredentials.enabled=true``) to the command.
+.. tip::
+
+    The API server deployed will be configured to use the hosting Kubernetes cluster to launch tasks by default. Refer to :ref:`sky-api-server-configure-credentials` to configure credentials for different clouds.
 
 .. tip::
```

Comment:
This tip box feels a bit unecessary. How about we just add a comment in the commands above before the line for `WEB_PASSWORD` to ask user to customize their password?